doc/reference/pipeline_spec.md

```diff
@@ -250,12 +283,9 @@ the commit itself) will be processed.
 
 Union inputs take the union of other inputs. For example:
 
-| inputA | inputB | inputA ∪ inputB |
-| ------ | ------ | --------------- |
-| foo    | fizz   | foo             |
-| bar    | buzz   | fizz            |
-|        |        | bar             |
-|        |        | buzz            |
+- inputA: foo, bar
```

Comment:
I did too. Unfortunately it doesn't render correctly in HTML.  You can check the current version to see or run `make html` from `docs`